common/src/main/kotlin/io/homeassistant/companion/android/common/data/HttpExceptionCallAdapterFactory.kt

```diff
@@ -0,0 +1,53 @@
+package io.homeassistant.companion.android.common.data
+
+import io.homeassistant.companion.android.common.exception.HttpException
```

Comment:
```suggestion
import io.homeassistant.companion.android.common.data.exception.HttpException
```
And you should also move `MalformedHttpUrlException`